ext/afform/core/xml/Menu/afform.xml

```diff
@@ -1,5 +1,10 @@
 <?xml version="1.0" encoding="iso-8859-1" ?>
 <menu>
+  <item>
+    <path>civicrm/afform/view</path>
```

Comment:
@totten just to clarify your questions about `server_route`, the heart of the PR is right here, where we add a new generic menu route that handles all afforms. Going to this route and appending the name of a form (e.g. `civicrm/afform/view/myCoolForm`) will load that form.

Reply:
Aaah, OK. That interpretation is clearer.